systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/KafkaBridgeExampleClients.java

```diff
@@ -9,24 +9,48 @@
 import io.strimzi.systemtest.Constants;
 import io.strimzi.systemtest.resources.KubernetesResource;
 import io.strimzi.systemtest.resources.ResourceManager;
-import io.strimzi.systemtest.resources.crd.KafkaClientsResource;
 
 import java.util.HashMap;
 import java.util.Map;
 
 // HTTP Bridge clients
-public class KafkaBridgeClientsResource extends KafkaClientsResource {
+public class KafkaBridgeExampleClients extends KafkaBasicExampleClients {
 
-    private final int port;
-    private final int pollInterval;
```

Comment:
Why it is not final? One of the advantages of the builder pattern is that the objects that are created are `immutable`.  Marking this as *not* final you saying that it can be changed...